### DISCUSSION TITLE
Doing the backup/restore script in-place of an existing install to upgrade from 0.15.12 -> 0.16.0

### DISCUSSION THREAD

**Discussion by bcredeur97** [amidaware/tacticalrmm#1594] [Q&A] [answered]
I was actually running TacticalRMM on Ubuntu 22.04 LTS and have been for quite some time, I actually completely missed that you were supposed to be running it on 20.04 with 0.15 and prior versions... anyways my instance lives so I guess it wasn't so bad

Now seeing the new database changes and such with 0.16.0; I'd like to get switched over to that, but I'm curious if I can do it "in-place" instead of standing up a brand new machine. 

Can I just simply do a backup, take a snapshot of my TacticalRMM VM for safety, and run the restore script and it will do everything, including the upgrade and doing the new database and everything (seeing how the restore scripts seems to take the place of the installation script in the docs)? Or is it more complicated than this? Do I need to do like an uninstallation prior to running the restore script or anything like that, or will that script handle doing the "full upgrade" of a pre-existing installation and not break things?

Just wanted to ask to be sure!

**Comment by wh1te909** [comment, marked as answer]
restore script needs to be run on a fresh instance with nothing installed on it, otherwise it straight up won't work. so you can snapshot, wipe the virtual hdd, reinstall ubuntu on the same vm and then run the restore script.

**Reply by bcredeur97** [reply]
OK awesome, that is what I will do, thanks Dan!
